sklift/datasets/descr/hillstrom.rst

```diff
@@ -42,4 +42,13 @@ Finally, we have a series of variables describing activity in the two weeks foll
 
 * Visit: 1/0 indicator, 1 = Customer visited website in the following two weeks.
 * Conversion: 1/0 indicator, 1 = Customer purchased merchandise in the following two weeks.
-* Spend: Actual dollars spent in the following two weeks.
\ No newline at end of file
+* Spend: Actual dollars spent in the following two weeks.
+
+Key figures
+################
+
+* Format: CSV
+* Size: 433KB (compressed) 4935KB (uncompressed)
+* Rows: 64000
```

Comment:
For convenience, it is better to divide thousands with separator, e.g. `64,000` vs `64000`